src/nvim/globals.h

```diff
@@ -1167,6 +1167,8 @@ EXTERN char_u e_dirnotf[] INIT(= N_(
 EXTERN char_u e_unsupportedoption[] INIT(= N_("E519: Option not supported"));
 EXTERN char_u e_fnametoolong[] INIT(= N_("E856: Filename too long"));
 EXTERN char_u e_float_as_string[] INIT(= N_("E806: using Float as a String"));
+EXTERN char_u e_autocmd_err[] INIT(=N_(
+    "E920: autocmd has thrown an exception: %s"));
```

Comment:
@bfredl This should be in the 5000+ range. 920 in Vim is channel-related.

Reply:
It used to be our range. (Jobstart etc) Or did we change our channel errors to be the in the same range as vim channel errors?

Reply:
Ah, if you chose it for reason, so be it. No big deal.

Reply:
Feel free to change it. But how do we know the other nvim specific codes in 900 won't collide? Or did vim-dev avoid individual numbers in 900 if not the entire range?

Reply:
@bfredl That's why we started using the 5000+ range, to avoid collisions.

Reply:
I meant the existing numbers. Do we know that vim-dev won't use them, or should they be changed before nvim 1.0?

Reply:
They should  be changed.